### DISCUSSION TITLE
MDN Offline Not Working on Android (Pixel 7)

### DISCUSSION THREAD

**Discussion by nickst0tt** [mdn/mdn-community#677] [MDN Plus]
When I enable offline access and select download, the unpacking gets about halfway before it just resets. Tried rebooting the device, changing connection method (WiFi / Cellular), tried in Chrome + Firefox natively and in the PWA for each browser respectively. Reset occurs between ~30% - 60% unpack progress. Works fine in Chrome on desktop.

**Comment by pransh15** [comment]
Hi @nickst0tt, thanks for posting this! 👋 
I can find an old issue on Chrome for Desktop(https://github.com/mdn/yari/issues/8022), but you cleared that it works fine for you there.

I haven't discovered any other issues linking to this, can you please submit a [new issue](https://github.com/mdn/yari/issues/new?assignees=&labels=bug&projects=&template=bug-report.yml) for this?

**Reply by nickst0tt** [reply]
Hi @pransh15, thanks for your reply. Unfortunately my subscription has lapsed as this was the only feature I was interested in.

I have a video of the issue but am unable to troubleshoot it further. Shall I still submit an issue?

https://github.com/mdn/mdn-community/assets/143878139/ffe2f18e-9a7a-451f-a07b-3715c9ffa39d

**Comment by rbryx** [comment]
iOS is affected too
